### PR TITLE
Vote-3147: RTL style updates

### PR DESCRIPTION
**Pull request by clmedders**
## Jira ticket

[Vote-3147](https://cm-jira.usa.gov/browse/VOTE-3147)

## Description

Addressing various RTL style issues:
1. accessibility page had element off to the side causing it to zoom out in mobile 
2. the arrows for quick links were not pointing the right direction 
3. the accessibility bar was going behind the touchpoints form on the accessibility page 

<details>
<summary>Screen shots</summary>
<br>
Screenshot 2024-12-01 at 8 45 21 PM
<br>
<br>
Screenshot 2024-12-01 at 8 44 50 PM
<br>
<br>
Screenshot 2024-12-01 at 8 44 27 PM
</details>

### Post-deploy steps

1. cd into the `votegov` theme and run `npm run build`

### QA/Testing instructions

1. switch to the AR translation and verify that the arrows for quick links are pointing the right direction 
2. visit the accessibility page  and ensure that you do not need to scroll over to see the page while in mobile view 
3. while on the accessibility page scroll to the touchpoints form and check that the accessability bar does not go behind the form.

## Checklist for the Developer

- [x] A link to the JIRA ticket has been included above.
- [x] No merge conflicts exist with the target branch.
- [x] Automated tests have passed on this PR.
- [x] A reviewer has been designated.
- [x] Deployment and testing steps have been documented above, if applicable.

## Checklist for the Peer Reviewers

- [x] The file changes are relevant to the task objective.
- [x] Code is readable and includes appropriate commenting.
- [x] Code standards and best practices are followed.
- [x] QA/Test steps were successfully completed, if applicable.
- [x] Applicable logs are free of errors.

### CODE REVIEW

File: web/themes/custom/votegov/src/sass/components/quick-links.scss [hunk 1]
```diff
@@ -48,8 +48,12 @@
     @include u-text('no-underline');
 
     @include after {
-      @include u-padding-left(2);
+      padding-inline-start: 1rem;
       background: url('../../img/svg/arrow-white.svg') no-repeat right center;
+      [dir="rtl"] & {
+      background: url('../../img/svg/arrow-white-rtl.svg') no-repeat left center;
```
Comment (clmedders) [inline web/themes/custom/votegov/src/sass/components/quick-links.scss:54]: i updated the current arrow svg to point left.. and added it to the RTL language

File: web/themes/custom/votegov/src/sass/uswds-overrides/usa-banner.scss [hunk 2]
```diff
@@ -48,6 +47,9 @@
 
 .usa-banner__header-text {
   color: var(--usa-banner-text);
+  a::before{
+    display: none;
+      }
```
Comment (rayestrada) [inline web/themes/custom/votegov/src/sass/uswds-overrides/usa-banner.scss:52]: I couldn't find this being used.

**Review by rayestrada — CHANGES_REQUESTED** [review]

File: web/themes/custom/votegov/src/sass/components/columns.scss [hunk 3]
```diff
@@ -4,6 +4,5 @@
 .vote-columns--two {
   @include at-media('tablet-lg') {
     column-count: 2;
-    column-gap: 2rem;
```
Comment (clmedders) [inline web/themes/custom/votegov/src/sass/components/columns.scss:7]: This was messing with the wrapping of the arrow for the quick links.. and when i removed it it did not adjust much so didn't seem like it was needed... let me know if you feel other wise

**Comment by rayestrada** [comment]
You were right about using a separate svg for rtl. Changing the pseudo element to inline block caused it to break to a new line by itself when the width of the column decreased.

**Pull request merged**